tornado_botocore/base.py

```diff
@@ -26,6 +27,7 @@ def __init__(self, service, operation, region_name, endpoint_url=None, session=N
             service, region_name=region_name, endpoint_url=endpoint_url)
         self.endpoint = self.client._endpoint
         self.operation = operation
+        AsyncHTTPClient.configure("tornado.curl_httpclient.CurlAsyncHTTPClient")
```

Comment:
Tornado states that `Proxies are currently only supported with curl_httpclient` http://www.tornadoweb.org/en/stable/httpclient.html#request-objects

And indeed I get a timeout if I don't set `CurlAsyncHTTPClient` as `httpclient`.